model/src/main/proto/interaction_object.proto

```diff
@@ -68,10 +68,15 @@ message ImageWithRegions {
     string label = 1;
 
     message Region {
+      // Represents the different Region Types.
+      enum RegionType {
+        RECTANGLE = 0;
```

Comment:
Proto enums should default to an unspecified value for the default (0) case, e.g.:

```proto
enum RegionType {
  UNSPECIFIED_REGION_TYPE = 0;
  RECTANGLE = 1;
}
```

This is a best practice to aid in maintaining backward compatibility.

Feel free to resolve this comment once it's fixed.

Reply:
done

Reply:
done